tmp/watch/0_Markdown-Cheatsheet.html

```diff
@@ -0,0 +1,471 @@
+<!DOCTYPE html>
```

Comment:
What's up with these cheatsheets?

Reply:
Also should't all these files in the `tmp` folder be in the `gitignore`?

Reply:
The Markdown was part of the fslab-docs template, it can be removed.
Was not sure about this, but it makes sense.